templates/tfengine/recipes/cicd.hcl

```diff
@@ -259,6 +259,32 @@ schema = {
               }
             }
           }
+          worker_pool = {
+            description          = <<EOF
+              Optional worker pool attribute. Required for CICD to work with private cluster endpoints.
+            EOF
+            type                 = "object"
+            additionalProperties = false
+            required = [
+              "project",
+              "location",
+              "name"
+            ]
+            properties = {
+              project = {
+                description = "The project worker pool belongs to."
```

Comment:
The project where the worker pool belongs.

Reply:
done